GameData/SPEngine/Parts/Aerobee_proc.cfg

```diff
@@ -1,4 +1,18 @@
-+PART[ROAerobeeSustainer]:AFTER[ZREALPLUME]
++PART[ROAerobeeSustainer]:NEEDS[!ROEngines]:AFTER[ZREALPLUME]
+{
+	@name = SPEngineWAC
+	@title = B-class Procedural Engine
+	@description = Small hypergolic sustainer engine, like Aerobee or very early AJ-10s.
+	MODULE
+	{
+		name = ModuleSPEngine
+		familyLetter = B
+	}
+	!identicalParts = // we're not identical to the part we copied!
+	!engineType = // prevent ROE from deleting us
+}
+
++PART[ROE-Aerobee]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
 {
 	@name = SPEngineWAC
 	@title = B-class Procedural Engine
```

Comment:
Again, description will want a "from ROEngines" attribution.  (Same applies to several more configs below.)